### PR TITLE
update reset script for pln-svc

### PR DESCRIPTION
**Pull request by ramessesii2**
update reset script for pln-svc
    - remove kcp-infra specific code
    - add code to remove tekton-chains and tekton-results controller
    - 
#### How to test
One can do a `./developer/openshift/reset.sh -h ` to see all the options available with reset script to scrap the deployed pipeline-service deployments.

Signed-off-by: Satyam Bhardwaj <sabhardw@redhat.com>

### CODE REVIEW

**Comment by ramessesii2** [comment]
FYI: #407 seems to resist deletion of Pipeline-Service argo cd application. Since, the same command - `argocd app delete pipeline-service --yes`  is used for the deletion of an argocd application. So, I'm hoping this would be resolved once we find the solution for the `OutOfSync` state of Pipeline-Service Argo CD application.

File: developer/openshift/reset.sh [hunk 1]
```diff
@@ -214,76 +210,21 @@ uninstall_operators(){
...
-    printf "\n  Uninstalling ckcp from the cluster:\n"
-    kubectl delete -k "$CKCP_DIR/openshift/overlays/dev" --ignore-not-found=true
-    # Check if the ckcp pod is removed from the compute cluster
-    mapfile -t ckcp_pod < <(kubectl get pods -n ckcp -l "app=kcp-in-a-pod" | cut -d " " -f 1 | tail -n +2)
-    if (( ${#ckcp_pod[@]} >= 1  )); then
-        printf "\n[ERROR] Couldn't remove the ckcp pods: %s ${ckcp_pod[*]}" >&2
+    # Checks if the operators are uninstalled successfully
+    mapfile -t controllers < <(kubectl get ns | grep -iE "tekton-results|tekton-chains|pipelines-as-code" | cut -d " " -f 1)
+    if (( ${#controllers[@]} >= 1 )); then
+        printf "\n[ERROR] Couldn't reomve all Controllers, please try removing them manually." >&2
```
Comment (Roming22) [inline developer/openshift/reset.sh:217]: reomve -> remove

Comment (ramessesii2) [reply developer/openshift/reset.sh]: done.

**Review by bnallapeta — COMMENTED** [review]
Tried running it locally, but could not get it to work as expected.

```
[bnr@bnr pipeline-service]$ ./developer/openshift/reset.sh --work-dir work --reset-hard -d

  Uninstalling Pipeline Service:

[ERROR] Couldn't uninstall Pipeline-Service Argo CD application.
```

**Comment by Roming22** [comment]
Let's test it again once #410 is merged.

**Comment by Mo3m3n** [comment]
We can also add the reset task to our CI (once all tests pass) this way we know when reset script breaks in the future.

**Comment by ramessesii2** [comment]
I've tested this reset script locally. I had to add commands to patch `pipeline-service` argocd application to remove finalizers which inhibits deletion.

**Review by Roming22 — APPROVED** [review]

**Pull request merged**